### ISSUE TITLE
FakeAuckland missing configruation and properties

### ISSUE THREAD

**Issue by nonhermitian** [Qiskit/qiskit-terra#8858]
### Environment

- **Qiskit Terra version**: 0.22rc1
- **Python version**: 3.10.6
- **Operating system**: 22.04

### What is happening?

`FakeAuckland` is missing config and properties

### How can we reproduce the issue?

```python
from qiskit.providers import fake_provider
backend = fake_provider.FakeAuckland()
backend.configuration()
```

### What should happen?

It should work like others do, eg

```python
backend = fake_provider.FakeHanoi()
backend.configuration()
```

**Comment by nonhermitian** [comment]
`FakePerth` is affected as well.  Hard to systematically parse though since blocked by #8859

**Comment by mtreinish** [comment]
Their `BackendV2` backends, which don't use configuration or properties, we decided to stop adding v1 backends for all the new fake backends. The migration guide between the 2 interfaces is here: https://qiskit.org/documentation/apidoc/providers.html#backendv1-backendv2 (there's also a new converter class in 0.22.0rc1 to wrap a v1 backend in a v2 interface, but you don't need that for fake backends as there are v2 instances available for all of them)

**Comment by nonhermitian** [comment]
Ahh I see. This is quite confusing from the user point of view though. In particular, Auckland, Hanoi, and Cairo were all released around the same time, and represent the same processor family. This makes it even harder to understand.

**Comment by nonhermitian** [comment]
Since this is "expected" behavior, closing in favor of #8861
